packages/plugin-legacy/src/index.ts

```diff
@@ -38,12 +38,9 @@ import {
 } from './snippets'
 
 // lazy load babel since it's not used during dev
-let babel: typeof import('@babel/core') | undefined
+let babel: Promise<typeof import('@babel/core')> | undefined
 async function loadBabel() {
-  if (!babel) {
-    babel = await import('@babel/core')
```

Comment:
I couldn't find a minimal repro. What I'm seeing locally was something like this:

```js
    if (!!babel.transform) {
      console.log("[ok]")
    } else {
      console.log("[not ok]", babel)
    }
```

```sh
$ pnpm -C playground/legacy build
...
[not ok] [Object: null prototype] { default: {} }
[not ok] [Object: null prototype] { default: {} }
[not ok] [Object: null prototype] { default: {} }
[not ok] [Object: null prototype] { default: {} }
[not ok] [Object: null prototype] { default: {} }
[ok]
[ok]
[ok]
```

I tried `JITI_DEBUG=true` but still don't have any clue about where this behavior is coming from.